crawl-ref/source/dat/des/variable/float.des

```diff
@@ -4901,6 +4901,33 @@ MAP
 .cccccccccc
 ENDMAP
 
+NAME:   dreamdust_wasp_nest
+TAGS:   no_monster_gen
+ORIENT: float
+DEPTH:  D:12-, Lair, Spider, Swamp, Depths
+WEIGHT: 5
+NSUBST: . = 6=1 / 6=1. / .
+: if you.in_branch("Depths") then
+SUBST:  x = b
+COLOUR: b = cyan
```

Comment:
two colours you want to avoid using for normal rock walls are cyan and lightcyan. In console, the former is used by metal walls and the latter is used by clear (transparent) rock and stone walls. Special wall colors aren't terribly necessary here to communicate the theme given the monster list and internal layout, and to a large extend the branch-default wall tiles will look better. If you're strongly attached to colored walls, you might consider what wizlab_cloud does: use `ETC_SILVER` crystal walls, which are cool since they change color with each UI update:
```
COLOUR:     b = silver
```
Probably as class as you can get to communicating "air-themed wall". This works for pretty much all locations. What I recommend doing is make the outer border be rock of the default coloring/tiling for the level, then have the internal structure be your `b` glyph, implying this is the area "built" by the wasps. Possible to also embed some `b` in the internal walls surrounding the central area, but ideally from the outside the vault looks relatively normal.